Simple-PHP-IPAM/assets/app.css

```
@@ -18,7 +18,68 @@
+  --font-size-xl:.9rem;
+  --font-size-2xl:.92rem;
+  --font-size-3xl:.95rem;
+  --font-size-4xl:1rem;
+  --font-size-5xl:1.05rem;
+  --font-size-6xl:1.15rem;
+  --font-size-7xl:1.35rem;
+  --font-size-8xl:1.6rem;
+  --font-size-9xl:1.9rem;
+
+  --focus-ring:0 0 0 2px var(--bg), 0 0 0 4px var(--link);
+  --font-mono:'Fira Code',ui-monospace,SFMono-Regular,Consolas,monospace;
```

Stylelint (17.7.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)